en/08_Changelogs/6.0.0.md

```diff
@@ -790,6 +790,14 @@ PHP code such as resolvers that were in [`silverstripe/asset-admin`](http://gith
 
 If your project does not have any custom GraphQL, after upgrading you may still have the old `.graphql-generated` and `public/_graphql` folders in your project. You can safely remove these folders.
 
+### `FormField` classes now use `FieldValidator` for validation
```

Comment:
Add this to the links list at the top of the page, and give it a short anchor

Reply:
Updated